### DISCUSSION TITLE
Welcome to skyland-beta Discussions!

### DISCUSSION THREAD

**Discussion by evanbowman** [evanbowman/skyland-beta#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Doronimmo** [comment]
Hello! This game is very fun! (8th place in leaderboard)
I'd love to share strategies etc, my favourite early game strat is cannon spam until I have enough money to upgrade to a nemesis.
Currently trying to beat the goblin raid challenge, help would be very much appreciated (one crew member is tough!)

**Reply by evanbowman** [reply]
Hi! Thanks for all your feedback so far, it's really helpful!

For the goblin raid challenge, upgrading one of your factories to a manufactory will let you build an arc-gun, which could be effective against the enemy's hull pattern. Another thing that could be helpful is building a couple of drone bays and deploying some repair drones. With a couple of repair drones placed, you should be able to cancel out the damage done by invading goblins to your power core, and slowly grind down the enemy with even a single cannon. I haven't tested that challenge level in a while, there are probably other solutions that I'm forgetting.
